great idea!

build fail looks real
```
======================================================================
FAIL: test_multiple_case_blocks_all_rebuilt (corehq.apps.users.tests.retire.RetireUserTestCase)
Rebuild all cases in forms with multiple case blocks
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/users/tests/retire.py", line 130, in test_multiple_case_blocks_all_rebuilt
    self.assertEqual(rebuild_case.call_count, len(case_ids))
AssertionError: 0 != 3
----------------------------------------------------------------------
```

--- a/corehq/apps/users/models.py
+++ b/corehq/apps/users/models.py
@@ -66,6 +66,7 @@ from dimagi.utils.django.database import get_unique_value
 from xml.etree import ElementTree
 
 from couchdbkit.exceptions import ResourceConflict, NoResultFound, BadValueError
+from dimagi.utils.parsing import json_format_datetime
 
 COUCH_USER_AUTOCREATED_STATUS = 'autocreated'
 
@@ -1634,7 +1635,7 @@ class CommCareUser(CouchUser, SingleMembershipMixin, CommCareMobileContactMixin)
     def retire(self):
         suffix = DELETED_SUFFIX
         deletion_id = random_hex()
-        deletion_date = datetime.utcnow()
+        deletion_date = json_format_datetime(datetime.utcnow())
         deleted_cases = set()
         # doc_type remains the same, since the views use base_doc instead
         if not self.base_doc.endswith(suffix):

--- a/corehq/apps/users/tests/retire.py
+++ b/corehq/apps/users/tests/retire.py
@@ -13,6 +13,13 @@ from corehq.form_processor.models import UserArchivedRebuild
 
 
 class RetireUserTestCase(TestCase):
+    dependent_apps = [
+        'corehq.apps.users', 'django_digest', 'auditcare', 'casexml.apps.case',
+        'corehq.form_processor', 'django.contrib.admin', 'corehq.apps.tzmigration',
+        'django_prbac', 'corehq.couchapps', 'tastypie', 'couchforms', 'corehq.apps.tour',
+        'corehq.apps.sms', 'corehq.apps.smsforms', 'corehq.apps.dropbox',
+        'corehq.apps.hqcase', 'touchforms.formplayer', 'corehq.sql_accessors'
+    ]
 
     def setUp(self):
         self.domain = 'test'

